src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -112,6 +118,34 @@ private function getIdentifiersFromItem($item): array
         $identifiers = [];
         $resourceClass = $this->getObjectClass($item);
 
+        $cacheKey = self::CACHE_KEY_PREFIX.md5(serialize([$resourceClass]));
```

Comment:
Indeed, copy paste you know :dancing_men: 

I made this in a hurry just to see if my first assumption was correct! I'll refactor this and decorate the `IriConverter` as suggested by @teohhanhui. Will be done this week!